frontend/src/components/CoursesFilter/components/CustomSelect/styled.ts

```diff
@@ -13,15 +12,16 @@ export const StyledSelect = styled(Select)({
   },
   '&.Mui-focused': {
     '& .MuiOutlinedInput-notchedOutline': {
-      borderColor: '#2C2525',
+      border: '1px solid #2C2525',
     },
   },
   '& .MuiSelect-select': {
     padding: '8px 14px',
     backgroundColor: '#7676801F',
   },
   '& .MuiOutlinedInput-notchedOutline': {
-    border: '2px solid #CBBEBE',
+    border: '1px solid #CBBEBE',
+    borderWidth: '1px',
```

Comment:
I guess it's an extra line, as it's already included in the `border` property

Reply:
agree

Reply:
👍